source/index.html.md.erb

```diff
@@ -57,11 +57,12 @@ These standards will enable us to maintain a consistent operating environment ac
 - [how to store and query logs](standards/logging.html)
 - [how to do penetration tests](standards/how-to-do-penetration-tests.html)
 - [how to manage access to your third-party service accounts](standards/accounts-with-third-parties.html)
+- [Use configuration management](standards/configuration-management.html)
```

Comment:
Is this configuration management line an artifact of a merge conflict? It seems like it shouldn't be part of this PR

Reply:
Yes, I think it's due to a merge conflict because this PR has been open for a while.